dev/CommonStyles/InteractionTests/CommonStylesTests.cs

```diff
@@ -147,5 +147,21 @@ public void RunCompactTests()
                 Verify.AreEqual(testResult, "Pass", "We expect compact test result is Pass"); // "Pass" string matches value used by MUXControlsTestApp.SimpleVerify
             }
         }
+
+        [TestMethod]
+        public void CornerRadiusTest()
+        {
+            if (!PlatformConfiguration.IsOsVersionGreaterThan(OSVersion.Redstone4))
```

Comment:
The test should still run down-level to verify that the conditional Xaml is working correctly. We want to make sure that the new styles work on all supported platforms.